src/app/shared/doc-viewer/doc-viewer.ts

```diff
@@ -60,11 +60,14 @@ export class DocViewer implements OnDestroy {
   }
 
   /**
-   * Updates the displayed document
-   * @param document The raw document content to show.
+   * Updates the displayed document.
+   * @param rawDocument The raw document content to show.
    */
-  private updateDocument(document: string) {
-    this._elementRef.nativeElement.innerHTML = document;
+  private updateDocument(rawDocument: string) {
+    // Replaces all hash base links with the current path
+    const correctedDocument = rawDocument.replace(
+      /(<a href="#)+/g, `<a href="${window.location.pathname}#`);
```

Comment:
Could you run this through `DomSanitizer.sanitize`?

Reply:
Done